### PR TITLE
database model in template.json

### PR DESCRIPTION
**Pull request by AlexisRoe**
Please look, mabye i made a mistake in the logic of the document.

#7 

--> 4YI
Function Sheets are documents used in hotels for handling events. They represent the neccessary data for an event for one day, could be a serial as well, so the employee have all to work on it. My template did not have all information from reality, but enough for my small project.

Thanks for all reviews or comments to that model.

### CODE REVIEW

**Review by lmachens — APPROVED** [review]
I am not sure what to review here :), the changes looks great 👍

**Comment by AlexisRoe** [comment]
> I am not sure what to review here :), the changes looks great 👍

@lmachens im not sure if this data-design is too complex or not logical at all, thats it... thanks for the reply 🤷‍♂️👍

File: template.json [hunk 1]
```diff
@@ -40,26 +40,26 @@
...
-          "food": ["strings"],
-          "beverages": ["strings"]
+          "food": ["string", "string", "string"],
+          "beverages": ["string", "string", "string"]
         },
         {
           "location": "string",
           "titel": "string",
           "pax": "number",
           "start": "time",
```
Comment (lmachens) [inline template.json:50]: How do you want to save `time` in JavaScript/JSON?
You could think about a UNIX-timestamp or an ISO-String.

**Review by lmachens — COMMENTED** [review]
Lgtm :-)

File: template.json (hunk 1 above)
Comment (AlexisRoe) [reply template.json:50]: i think the unix timestamp is more useful, it holds time and date so i could spare room

File: template.json [hunk 2]
```diff
@@ -40,26 +40,26 @@
...
           "end": "time",
-          "food": ["strings"],
-          "beverages": ["strings"]
+          "food": ["string", "string", "string"],
```
[the remark this answers is not in the document]
Comment (AlexisRoe) [reply template.json:52]: 🤦‍♂️ thanks for correcting the typos

**Comment by AlexisRoe** [comment]
@lmachens thanks a lot

**Pull request merged**